tests/jballerina-unit-test/src/test/resources/test-src/statements/variabledef/global_variable_init_with_onfail_negative.bal

```diff
@@ -1,39 +0,0 @@
-// Copyright (c) 2023 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
-//
-// WSO2 Inc. licenses this file to you under the Apache License,
-// Version 2.0 (the "License"); you may not use this file except
-// in compliance with the License.
-// You may obtain a copy of the License at
-//
-// http://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing,
-// software distributed under the License is distributed on an
-// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
-// KIND, either express or implied.  See the License for the
-// specific language governing permissions and limitations
-// under the License.
-
-int i;
```

Comment:
If we revert this PR, we will be allowing these kinds of scenarios in the 4.1 release. Although it is acceptable because we intend to permit scenarios that were previously disallowed, my concern is about when we will disallow them again in a future version. Will it be in version 4.2 or 5.0? If we make this change in version 4.2, it will be a breaking change that affects version 4.1.